tests/python_client/testcases/test_query.py

```diff
@@ -4135,3 +4135,357 @@ def test_query_iterator_expr_empty_with_random_pk_pagination(self, batch_size, o
+                                                             default_value_fields={ct.default_float_field_name: np.float32(10.0)})[0:2]
+        # 2. create index on vector field and load
+        index = "HNSW"
+        params = cf.get_index_params_params(index)
+        default_index = {"index_type": index, "params": params, "metric_type": "COSINE"}
+        vector_name_list = cf.extract_vector_field_name_list(collection_w)
+        vector_name_list.append(ct.default_float_vec_field_name)
+        for vector_name in vector_name_list:
+            collection_w.create_index(vector_name, default_index)
+        # 3. create index on scalar field with None data
+        scalar_index_params = {"index_type": varchar_scalar_index, "params": {}}
+        collection_w.create_index(ct.default_string_field_name, scalar_index_params)
+        # 4. create index on scalar field with default data
+        scalar_index_params = {"index_type": numeric_scalar_index, "params": {}}
+        collection_w.create_index(ct.default_float_field_name, scalar_index_params)
+        collection_w.load()
+        pos = 5
+        int64_values, float_values = [], []
+        scalar_fields = vectors[0]
+        for i in range(pos):
+            int64_values.append(scalar_fields[0][i])
+            float_values.append(scalar_fields[5][i])
+        res = [{ct.default_int64_field_name: int64_values[i], default_float_field_name: float_values[i]} for i in range(pos)]
+
+        term_expr = f'{ct.default_int64_field_name} in {int64_values[:pos]}'
+        # 5. query
+        collection_w.query(term_expr, output_fields=[ct.default_int64_field_name, ct.default_float_field_name],
+                           check_task=CheckTasks.check_query_results, check_items={exp_res: res})
+
+    @pytest.mark.tags(CaseLabel.L1)
+    def test_query_iterator_with_none_data(self, null_data_percent):
+        """
+        target: test query iterator normal with none data
+        method: 1. query iterator
+                2. check the result, expect pk
+        expected: query successfully
+        """
+        # 1. initialize with data
+        batch_size = 100
+        collection_w = self.init_collection_general(prefix, True, is_index=False,
+                                                    nullable_fields={ct.default_string_field_name: null_data_percent})[0]
+        collection_w.create_index(ct.default_float_vec_field_name, {"metric_type": "L2"})
+        collection_w.load()
+        # 2. search iterator
+        expr = "int64 >= 0"
+        collection_w.query_iterator(batch_size, expr=expr,
+                                    check_task=CheckTasks.check_query_iterator,
+                                    check_items={"count": ct.default_nb,
+                                                 "batch_size": batch_size})
+
+    @pytest.mark.tags(CaseLabel.L1)
+    def test_query_normal_none_data_partition_key(self, enable_dynamic_field, null_data_percent):
```

Comment:
This case may block the CI if issue #36213 is fixed, so here may add skip fixture.

Reply:
add skip done